src/index.js

```diff
@@ -63,11 +79,11 @@ program.action(function(cmd, cmd2) {
       "instead"
     );
   } else {
-    var suggestion = didYouMean(cmd, commandNames);
-    suggestion = suggestion || didYouMean([cmd, cmd2].join(":"), commandNames);
-    if (suggestion) {
-      logger.error();
-      logger.error("Did you mean", clc.bold(suggestion) + "?");
+    // Check if the first argument is close to a command.
+    if (!suggestCommands(cmd, commandNames)) {
+      // Check to see if combining the two arguments comes close to a command.
+      // e.g. `firebase hosting disable` may suggest `hosting:disable`.
+      suggestCommands([cmd, cmd2].join(":"), commandNames);
```

Comment:
is `cmd2` always guaranteed to exist? If so, perhaps we can put this extra check into the suggestCommands method so that it takes care of the whole command validation logic.

Reply:
Discussed offline - went for making `suggestCommand` logic clean and understandable by not moving in the extra check into that function.